int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -278,16 +278,19 @@ countryDict.push({ key: "AF", value: "AFG" },
 	{ key: "ZW", value: "ZWE" });
 function countryAlpha3(incomingCountryIso2) { 
 	var iso2ToIso3CountryDict;
-	countryDict.forEach((val) => iso2ToIso3CountryDict = (incomingCountryIso2.match(val.key)) ? val.value : iso2ToIso3CountryDict);
-	return iso2ToIso3CountryDict || vDefaultCountry;
+    for (var i = 0; i < countryDict.length; i++) {
```

Comment:
Don't use C style loops here (especially since the logic you're implementing doesn't require a C style loop). In this particular case, since the issue is specific to a browser, let's use the [polyfill](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach)